### PR TITLE
feat: Add 500 error page

### LINKED ISSUE #87: Handle 500 errors
500 errors should be handled to not display the default 500 error Nuxt page, for example:

https://aescan.dev.aepps.com/transactions/th_QRAjvukXobteRXCFzZZtA23iNJTiqpX5LxZN4nhdD6vUPG96X

We need a 500 Internal Server Error page, skip illustration for now.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #87 

## Demo
image
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] Add error page

### CODE REVIEW

**Comment by janmichek** [comment]
Few comments on this:

1) Hopefully, we don't have any issues causing 500 errors on production .)
The way I was developing it was to set `dev: false` on nuxt config. Then I removed some imports to cause an error.

2) the visual emoticons have not been implemented as discussed.

3) By the visual material there should be the title `Not found` but I don't think it's correct. There should be an `Internal server error` or `Unexpected Error`

4) The previous error pages have left alignment and here was the center. So I propose to keep it consistent as shown in the demo.

File: src/error.vue [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+      property="og:url"
+      :content="APP_URL"/>
+    <Meta
+      name="twitter:card"
+      content="summary_large_image"/>
+    <Meta
+      name="twitter:site"
+      :content="APP_CREATOR"/>
+    <Meta
+      name="twitter:creator"
```
Comment (janmichek) [inline src/error.vue:63]: these tags has not been reused from app.vue as well as the layout

**Review by lukeromanowicz — APPROVED** [review]
Looking really nice! well done =)

File: src/error.vue [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+  <the-header/>
+  <div class="error">
+    <div class="error__parallax">
+      <page-header>
+        Unexpected Error
+      </page-header>
+
+      <app-panel class="error__panel">
+        <h1 class="error__heading">
```
Comment (michele-franchi) [inline src/error.vue:73]: Now that I see it live I would change it to h2, to me it looks too big and also does not have a hierarchy.
If you agree let's also add a comment to the design.

File: src/error.vue [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+<script setup>
+import { useHead } from '@vueuse/head'
+import AppLink from '@/components/AppLink'
+import PageHeader from '@/components/PageHeader'
+import AppButton from '@/components/AppButton'
+import AppPanel from '@/components/AppPanel'
+import TheHeader from '@/components/TheHeader'
+import TheFooter from '@/components/TheFooter'
+import { APP_CREATOR, APP_DESCRIPTION, APP_KEYWORDS, APP_TITLE, APP_URL } from '~/utils/constants'
```
Comment (michele-franchi) [inline src/error.vue:109]: Different import? Can you unify it?

**Review by michele-franchi — CHANGES_REQUESTED** [review]
It looks good and it's according to the design but I would slightly adjust it.

File: src/error.vue (hunk 2 above)
Comment (janmichek) [reply src/error.vue]: fixed

File: src/error.vue (hunk 3 above)
Comment (janmichek) [reply src/error.vue]: fixed

File: src/error.vue (hunk 2 above)
Comment (janmichek) [reply src/error.vue]: image

**Review by michele-franchi — APPROVED** [review]
Good job

**Pull request merged**